### DISCUSSION TITLE
Problem with the file recovery window

### DISCUSSION THREAD

**Discussion by Kaos0019** [folke/noice.nvim#646] [Q&A]
Recently I encountered a problem: it is impossible to restore the swap file due to the fact that everything is directed to the notification window and to a small window open at the bottom of the screen. In none of these places can I pass the number indicating the number of the swap file, so I cannot restore the file I need. I tried disabling the plugin and everything worked without problems

**Comment by adoyle-h** [comment]
Same problem with noice.nvim v2.0.1 and neovim v0.9.4. @folke Can you help me answer that?
